lib/jekyll-relative-links.rb

```diff
@@ -1,6 +1,10 @@
 require "jekyll"
-require "jekyll-relative-links/generator"
+require "jekyll-relative-links/hook"
 require "jekyll-relative-links/context"
 
 module JekyllRelativeLinks
 end
+
+Jekyll::Hooks.register :site, :post_render do |site|
```

Comment:
One suggestion would be to run this hook for each document as it's converted, rather than all at once for the site.